src/Article.js

```diff
@@ -24,6 +19,7 @@ class Article extends Component {
               <h3>{this.props.title}</h3>
               <h6>{this.props.category}</h6>
               <span className="articleDate">Published {this.props.date}</span>
+              {/* It's probably better to format the date here in article instead of in the parent */}
```

Comment:
It's generally a good idea to do all your data formatting either as early or as late as possible. Keeping your data consistent in the middle makes it easier to debug little issues and keep track of what the data looks like at certain points.